ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -230,7 +236,227 @@
 	return nil
 }
 
+func (s *LifeCycleServiceServer) sshFunc() error {
+	config := &ssh.ClientConfig{
+		User: "root",
+		Auth: []ssh.AuthMethod{
+			ssh.Password(""),
+		},
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
```

Comment:
so this should be dealt with later with PSE - on secure way to do this.... 
doing a "trust on first connect" isn't great - since container is ephemeral .... 

I will dismiss the alert but this will need to be brought back up later